src/BottomNavigation/BottomNavigationButton.js

```diff
@@ -0,0 +1,117 @@
+// @flow weak
+
+import React, { Component, PropTypes, cloneElement, isValidElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import ButtonBase from '../internal/ButtonBase';
+
+export const styleSheet = createStyleSheet('BottomNavigationButton', (theme) => {
+  const { palette, typography, transitions } = theme;
+
+  return {
+    root: {
+      transition: `${transitions.create('color', '250ms')}, ${transitions.create('padding-top', '250ms')}`,
+      paddingTop: 8,
+      paddingBottom: 10,
+      paddingLeft: 12,
+      paddingRight: 12,
+      minWidth: 80,
+      maxWidth: 168,
+      background: 'none',
+      color: palette.text.secondary,
+      flex: '1',
+    },
+    selected: {
+      paddingTop: 6,
+      color: palette.primary[500],
+    },
+    selectedIconOnly: {
+      paddingTop: 16,
+    },
+    label: {
+      fontSize: 12,
+      opacity: 1,
+      transition: 'font-size 0.2s, opacity 0.2s',
+      transitionDelay: '0.1s',
+    },
+    selectedLabel: {
+      fontSize: typography.fontSize,
+    },
+    hiddenLabel: {
+      opacity: 0,
+      transitionDelay: '0s',
+    },
+    icon: {
+      display: 'block',
+    },
+  };
+});
+
+export default class BottomNavigationButton extends Component {
+  handleChangeIndex = () => {
+    const { onChangeIndex, index } = this.props;
+    if (onChangeIndex) {
+      onChangeIndex(index);
+    }
+  };
+
+  render() {
+    const {
+      label,
+      icon: iconProp,
+      selected,
+      className: classNameProp,
+      showLabel: showLabelProp,
+      onChangeIndex, // eslint-disable-line no-unused-vars
+      index, // eslint-disable-line no-unused-vars
+      ...other,
+    } = this.props;
+    const classes = this.context.styleManager.render(styleSheet);
+    const className = classNames(classes.root, {
+      [classes.selected]: selected,
+      [classes.selectedIconOnly]: !showLabelProp && !selected,
+    }, classNameProp);
+    const classNameIcon = classNames(classes.icon,
+      isValidElement(iconProp) ? iconProp.props.className : null);
+    const classNameLabel = classNames(classes.label, {
+      [classes.selectedLabel]: selected,
+      [classes.hiddenLabel]: !showLabelProp && !selected,
+    });
+    const icon = isValidElement(iconProp) ?
+      cloneElement(iconProp, { className: classNameIcon }) :
+        <span className="material-icons">{iconProp}</span>;
+
+    return (
+      <ButtonBase className={className} {...other} onClick={this.handleChangeIndex}>
```

Comment:
`focusRipple` isn't applied by default. I think that we should add it here. To get this when focused.
![capture d ecran 2016-11-09 a 21 36 07](https://cloud.githubusercontent.com/assets/3165635/20153765/9118c588-a6c4-11e6-8c6b-933eddf12cbd.png)

@nathanmarks Shouldn't it be the default value?